community/bolt/src/main/java/org/neo4j/bolt/v1/messaging/PackStreamMessageFormatV1.java

```diff
@@ -108,7 +108,8 @@ public void handleRunMessage( String statement, Map<String,Object> params )
         {
             packer.packStructHeader( 2, MessageTypes.MSG_RUN );
             packer.pack( statement );
-            packer.packRawMap( params );
+            packer.packParameterMap( params );
```

Comment:
That is how it worked before but then we must do another separate pass through the parameters